01_stratified_classification/nbconverted/plot_univariate_fs_results.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Analysis of feature selection results
+# 
+# We tried a few different feature selection methods for mutation prediction:
+# 
+# * mean absolute deviation
+# * pan-cancer univariate f-statistic (on training data, separate samples into mutated/not mutated and do an f-test)
+# * median univariate f-statistic across cancer types (same as above for each individual cancer type, then take median)
+# * MAD of univariate f-statistic across cancer types (same as above, but look for least variable genes)
+# 
+# Here, we're just looking at stratified CV across all cancer types, as a sanity check - the idea is to make sure the univariate correlation-based feature selection methods perform reasonably well. This isn't necessarily where we'd expect them to help performance, but we still don't want them to _hurt_ it.
```

Comment:
Very good intro, helps me (and probably future you) know exactly what the motivation is.